### PR TITLE
added Envs to influxdb

### PR DESCRIPTION
**Pull request by erSitzt**
We are using Docker with Mesos / Marathon and currently the metrics sent to influxdb do not contain the hostname on which the container (or cadvisor) is running.

You need to whitelist the Env-vars you want to be send to influxdb with --docker_env_metadata_whitelist

I'm new to Go , so this might not be the best/correct way to pass the env vars... but it works for me.

### CODE REVIEW

**Comment by orientzc** [comment]
Sorry. i use docker_env_metadata_whitelist parameter to get envs in influxdb. but it is working in cadvisor, not working in influxdb. the detail is the issue:#1559. could anyone help me? thanks.
ps: i try google/cadvisor:latest and google/cadvisor:canary.

**Comment by erSitzt** [comment]
@orientzc its not working because the pull-request was never merged...

**Comment by erSitzt** [comment]
@timstclair could someone review and merge this ? do have to do anything else ?

**Comment by orientzc** [comment]
@erSitzt Thanks very very much. 
i am sorry to be late. and using your Github Repository, will i only see envs in cpu_usage_per_cpu? is this because that you only edit the code of cpu_usage_per_cpu? and i will focus on the issue: #1427. and i expect good news. If  you send a message to me when the merge request handled, i will be lucky. good luck to you and me.
Thanks very very much.

**Comment by orientzc** [comment]
oh,  i close the issue: #1559. and should i send "ok to test" ?

File: storage/influxdb/influxdb.go [hunk 1]
```diff
@@ -163,6 +163,7 @@ func (self *influxdbStorage) tagPoints(ref info.ContainerReference, stats *info.
 		// merge with existing tags if any
 		addTagsToPoint(points[i], commonTags)
 		addTagsToPoint(points[i], ref.Labels)
+		addTagsToPoint(points[i], ref.Envs)
```
Comment (timstclair) [inline storage/influxdb/influxdb.go]: What happens if there's overlap in env & label & common tag names? We work around this in the prometheus plugin by prefiixing the keys: [metrics/prometheus.go](https://github.com/google/cadvisor/blob/5e78498404b49fae5bd0e073a44f9f8d875a6778/metrics/prometheus.go#L568)

**Comment by timstclair** [comment]
ok to test

**Comment by jcmcken** [comment]
Any updates on this?

**Comment by dashpole** [comment]
@Bart-Z 
The author hasn't responded in a while.  I'll close this for now.
Feel free to open a similar PR if you need this functionality, and I can review it.

**Pull request closed without merging**